src/cli/discord_cli.py

```
@@ -0,0 +1,278 @@
+            console.print(f"[green]✅ Discord bot '{instance_name}' started successfully![/green]")
+            console.print("[dim]The bot will continue running in the background.[/dim]")
+            track_command("discord_start", success=True, instance_name=instance_name, duration_ms=(time.time() - start_time) * 1000)
+        else:
+            console.print(f"[red]❌ Failed to start Discord bot '{instance_name}'[/red]")
+            track_command("discord_start", success=False, instance_name=instance_name, duration_ms=(time.time() - start_time) * 1000)
+            raise typer.Exit(1)
+            
+    except Exception as e:
+        console.print(f"[red]Error: {e}[/red]")
+        track_command("discord_start", success=False, error=str(e), duration_ms=(time.time() - start_time) * 1000)
+        raise typer.Exit(1)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)